python/equistore-operations/equistore/operations/_utils.py

```diff
@@ -179,7 +183,7 @@ def _check_same_gradients(
     a: TensorBlock,
     b: TensorBlock,
     fname: str,
-    check: Sequence[str] = ("samples", "components", "properties"),
+    check: List[str] = ("samples", "components", "properties"),
```

Comment:
I would change this back to a `Tuple`. Also we also have to change this in `equal_metadata.py`

Reply:
I think TorchScript checked the number of tuple elements in the type; i.e. `Tuple[str, str, str]` does not match `Tuple[str, str]`. And it does not supports the syntax for arbitrary sized tuple `Tuple[str, ...]`

Reply:
So we might actually have to change this back to using a special value `None`/`"all"`/... to check everything. Maybe `check: Union[List[str], str] = "all",`, which can be set to `"all"`, `"samples"`, `"components"`, `"properties"` or a list containing any of the last three.

Reply:
You are right, I did not test the most recent change. Agree with `None/"all"` solution. No strong opinions if we do "all" or None.